readme.md

```diff
@@ -75,6 +78,25 @@ dice =
 | "5"
 | "6"
 ```
+### Character Range
+
+Matching the output of a dice like in the previous example is pretty annoying.
+That's why the character range operator exists
+
+```py
+character_range = character ".." character : matches any character from ascii code of the first to the second character
+```
+
+#### What is a character
+
+The charater symbol in sebnf is only used for the character range
+
+Since the character range only accepts characters you don't need to escape them
```

Comment:
```suggestion
Since the character range only accepts characters you don't need to escape them.

```
Add punctuation and split off from next paragraph properly